src/js/mep-player.js

```diff
@@ -1140,7 +1140,7 @@ function constrainedSeekTo(player, media, targetTime) {
 				railWidth = 0,
 				rail = t.controls.find('.mejs-time-rail'),
 				total = t.controls.find('.mejs-time-total'),
-				others = rail.siblings(),
+				others = rail.siblings().filter(':visible'), // don't include `display: none` controls when sizing
```

Comment:
Good catch, but the issue is actually on the next line, 1144, where `lastControl` is set to `others.last()`. In our case, this is not a visible element, and throws off the calculations which try to reduce the rail size. Maybe I can change that line to something like `lastControl = others.filter(':visible').last()` instead to make the fix clearer.